Zend/zend_closures.c

```diff
@@ -160,15 +160,21 @@ ZEND_METHOD(Closure, call)
 		/* copied upon generator creation */
 		GC_DELREF(&closure->std);
 	} else {
+		zend_closure *fake_closure;
```

Comment:
Generally looks good, just wondering why this isn't simply stack allocated now?

Reply:
> Generally looks good, just wondering why this isn't simply stack allocated now?

He-he :) This was your commit b576bb901eb. As I remember, zend_bailout() might unwind C stack and make dangling pointer.